bpf/bpf_lxc.c

```diff
@@ -1072,7 +1071,6 @@ int tail_ipv4_policy(struct __sk_buff *skb)
 		ret = skb_redirect_to_proxy(skb, proxy_port);
 	}
 
-	skb->cb[0] = skb->mark; // essential for proxy ingress, see bpf_ipsec.c
```

Comment:
Do we have guarantees that cb[0] does not contain garbage when read by the bpf_ipsec.c if we only set it when we want to redirect to proxy?

Reply:
Perhaps not enough. I can try simplifying this change.